### ISSUE TITLE
Error in [conda install xformers -c xformers/label/dev]

### ISSUE THREAD

**Issue by 11whitewater** [facebookresearch/xformers#1084]
# 🐛 Bug

## Command
conda install xformers -c xformers/label/dev
## To Reproduce

1.
2.
3.

## Environment

Please copy and paste the output from the
environment collection script from PyTorch
(or fill out the checklist below manually).

You can run the script with:

```bash
# For security purposes, please check the contents of collect_env.py before running it.
python -m torch.utils.collect_env
```

- PyTorch Version (e.g., 1.0):
- OS (e.g., Linux):
- How you installed PyTorch (`conda`, `pip`, source):
- Build command you used (if compiling from source):
- Python version:
- CUDA/cuDNN version:
- GPU models and configuration:
- Any other relevant information:

## Additional context

![image](https://github.com/user-attachments/assets/de767e21-9e58-4c7a-a51a-7a5ce694d029)
![image](https://github.com/user-attachments/assets/a8716a2a-385f-45f7-984f-24e41cc96ae6)

**Comment by 11whitewater** [comment]
good heart people help help me😢

**Comment by lw** [comment]
Any specific reason you need the `dev` label? Could you use a stable version instead?

**Comment by 11whitewater** [comment]
> Any specific reason you need the `dev` label? Could you use a stable version instead?

hi, thanks for reply. I want  to train [StableSR] project, and it needs pytorch1.12.1+cuda11.7 .  StableSR Official recommended use [conda install xformers -c xformers/label/dev] to install xformers,  but i failed. I also tried [pip install xformers==0.0.16/0.0.16rc425],
but it's not compatible. Could you tell me which version of xformers will work with Pytorch 1.12.1+cuda11.7 ?

**Comment by lw** [comment]
I don't know off the top of my head, but PyTorch 1.12 is quite ancient, I don't know if any of our current packages is still compatible with that. I'd say that your best bet is to just ask conda to find out for you: if you ask it to install `pytorch==1.12.1 pytorch-mutex==1.0=cuda pytorch-cuda==11.7 xformers` then it should try to find a version of xFormers that works with the given requirements.

**Comment by lw** [comment]
BTW I ran that command and it looks like xFormers 0.0.22 should work for you.

**Comment by 11whitewater** [comment]
> BTW I ran that command and it looks like xFormers 0.0.22 should work for you.

Very lucky to have your help! I'll try again later!
